sirepo/package_data/static/js/raydata.js

```diff
@@ -488,6 +499,26 @@ SIREPO.app.directive('scansTable', function() {
                 $scope.saveColumnChanges();
             };
 
+            $scope.downloadSelectedAnalyses = () => {
+                if (! $scope.pdfSelectedScans.length) {
```

Comment:
This check can be removed once we only show the button to download once scans are selected.

Reply:
34d5c6d1b0aa5be7fb841e47a6fbf0fc34ecd9e9